dbms/src/Storages/DeltaMerge/tests/gtest_dm_file.cpp

```diff
@@ -156,6 +155,21 @@ class DMFileTest
 
     Context & dbContext() { return *db_context; }
 
+    static void checkMergedFile(
+        const DMFilePtr & dmfile,
+        UInt64 merged_number,
+        const std::set<String> & not_uploaded_files,
+        std::set<String> & checked_fnames);
+    static void checkColumnStats(const DMFilePtr & dmfile1, const DMFilePtr & dmfile2);
+
+    static void breakFileMetaV2File(const DMFilePtr & dmfile)
+    {
+        PosixWritableFile file(dmfile->metav2Path(), false, -1, 0666);
+        String s = "hello";
+        auto n = file.pwrite(s.data(), s.size(), 0);
+        ASSERT_EQ(n, s.size());
+    }
```

Comment:
Declare as a static function of `DMFileTest `. Because `DMFileTest ` is friend class of `DMFile`, so it can access to the private members of `DMFIle`.